intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/formatter/YangFormattingModelBuilder.java

```diff
@@ -0,0 +1,36 @@
+package tech.pantheon.yanginator.plugin.formatter;
+
+import com.intellij.formatting.Alignment;
+import com.intellij.formatting.FormattingContext;
+import com.intellij.formatting.FormattingModel;
+import com.intellij.formatting.FormattingModelBuilder;
+import com.intellij.formatting.FormattingModelProvider;
+import com.intellij.formatting.SpacingBuilder;
+import com.intellij.formatting.Wrap;
+import com.intellij.formatting.WrapType;
+import com.intellij.psi.codeStyle.CodeStyleSettings;
+import org.jetbrains.annotations.NotNull;
+import tech.pantheon.yanginator.plugin.YangLanguage;
+import tech.pantheon.yanginator.plugin.psi.YangTypes;
+
+public class YangFormattingModelBuilder implements FormattingModelBuilder {
+    private static SpacingBuilder createSpaceBuilder(final CodeStyleSettings settings) {
```

Comment:
Could you please put this private method below the public method ? When looking at a class, first thing you want to see is its public API, not internal details.